partner_compassion/tests/test_partner_compassion.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Samuel Fringeli <samuel.fringeli@me.com>
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo.tests.common import TransactionCase
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class TestMessages(TransactionCase):
+    def setUp(self):
+        super(TestMessages, self).setUp()
+
+        res_partner = self.env['res.partner']
+        self.church = res_partner.browse(8)
+        self.church.is_church = True
+
+        custom_vals = {
+            u'preferred_name': u'Samuel',
+            u'street': u'Impasse des Fr\xeanes 1',
+            u'zip': u'1669',
+            u'firstname': u'Samuel',
+            u'name': u'Fringeli Samuel',
+            u'city': u'Les Sciernes',
+            u'lastname': u'Fringeli',
+            u'ref': u'Fringeli',
+            u'church_id': self.church.id,
+            u'lang': u'en_US',
+            u'phone': u'+41 78 813 12 36'
```

Comment:
Do you really want to put your information on Github? I think random data should be used...

Reply:
Other people do it too ;)
https://github.com/CompassionCH/compassion-switzerland/blob/2845f2d19aa3be19680b9778dbca8efc07b372a1/wordpress_connector/models/contracts.py#L60-L71

Reply:
:laughing: It's my secret plan to become famous!
@samuel-fringeli Are you ok with this? If good, I will merge.